eng/build.ps1

```diff
@@ -182,30 +214,91 @@ function BuildSolution() {
         /p:QuietRestore=$quietRestore `
         /p:QuietRestoreBinaryLog=$binaryLog `
         /p:TestTargetFrameworks=$testTargetFrameworks `
+        /p:VisualStudioDropName=$vsDropName `
+        /p:IbcOptimizationDataDir=$IbcOptimizationDataDir `
         /p:TreatWarningsAsErrors=true `
         $suppressExtensionDeployment `
         @properties
 }
 
+function Restore-OptProfData() {
```

Comment:
Once we validate it's working I'll move this to Arcade so that msbuild and other repos can use it.